R/plot.data_table_threads_benchmark.R

```diff
@@ -23,39 +23,49 @@
 
 plot.data_table_threads_benchmark <- function(x, ...)
 {
-  df <- x
-  rownames(df) <- NULL
-  df$speedup <- df$medianTime[df$threadCount == 1] / df$medianTime
+  dt <- as.data.table(x)
```

Comment:
I don't think this line is necessary because x is already a data.table right?
also when you do a chain like this please avoid repeating dt:
```r
x[, `:=`(
  speedup = foo,
  type = "bar"
)]
```

Reply:
> I don't think this line is necessary because x is already a data.table right?
 also when you do a chain like this please avoid repeating dt:
> 
> ```r
> x[, `:=`(
>   speedup = foo,
>   type = "bar"
> )]
> ```

You're right, and done!